include/zephyr/bluetooth/services/ias.h

```diff
@@ -60,6 +68,60 @@ int bt_ias_local_alert_stop(void);
 #define BT_IAS_CB_DEFINE(_name)                                                                    \
 	static const STRUCT_SECTION_ITERABLE(bt_ias_cb, _CONCAT(bt_ias_cb_, _name))
 
+struct bt_ias_client_cb {
+	/** @brief Callback function for bt_ias_discover.
+	 *
+	 *  This callback is called when discovery procedure is complete.
+	 *
+	 *  @param conn Bluetooth connection object.
+	 *  @param err 0 on success, ATT error or negative errno otherwise
+	 */
+	void (*discover)(struct bt_conn *conn, int err);
+};
+
+/** @brief Set high alert level
+ *
+ *  @param conn Bluetooth connection object
+ *
+ *  @return Zero in case of success and error code in case of error.
+ *
+ */
+int bt_ias_client_alert_high(struct bt_conn *conn);
+
+/** @brief Set mild alert level
+ *
+ *  @param conn Bluetooth connection object
+ *
+ *  @return Zero in case of success and error code in case of error.
+ *
+ */
+int bt_ias_client_alert_mild(struct bt_conn *conn);
+
+/** @brief Set stop alert level
+ *
+ *  @param conn Bluetooth connection object
+ *
+ *  @return Zero in case of success and error code in case of error.
+ *
+ */
```

Comment:
Minor:

```suggestion
 *  @return Zero in case of success and error code in case of error.
 */
```

Applies multiple places

Reply:
Done